### DISCUSSION TITLE
WebDAV not working

### DISCUSSION THREAD

**Discussion by Hi-there-how-are-u** [sircharlo/meeting-media-manager#444] [Q&A] [answered]
Hi!
Suddenly my Webdav from box.com stopped working. So i decided to try 4shared instead. That does not work either for me so I tried Koofr and same here. Most likely I am doing something wrong but I do not know what.
In all these three providers I can easily log in to my account  but now with Webdav in MMM.

Does anyone else have these problems with these providers?
Or do you use something else? I am no idea what to try next... :)
All these three providers looks like this:
![bild](https://user-images.githubusercontent.com/36486218/186246267-31fd453d-140c-4285-a6c2-310412d6a86e.png)

Have a great day!

**Comment by sircharlo** [comment, marked as answer]
I just tested, my own WebDAV server works, along with my Koofr, PowerFolder and 4shared logins. Box.com, however, seems to have stopped working for some reason, but as stated [here](https://support.box.com/hc/en-us/articles/360043696414-WebDAV-with-Box)... No big surprise if the Box.com integration behaves a bit unpredictably.

If you're having issues, try changing your passwords perhaps?
